tests/test_tensorflow_data_api.py

```diff
@@ -27,138 +29,137 @@
 BATCH_SIZE = 32
 ROWS = 1000
 
-# We test for 2d, 3d, 4d and 5d data
-INPUT_SHAPES = [(10,), (10, 3), (10, 10, 3), (10, 10, 10, 3)]
-
 
-def ingest_in_tiledb(data: np.array, batch_size: int, uri: str):
-    dims = [
-        tiledb.Dim(
-            name="dim_" + str(dim),
-            domain=(0, data.shape[dim] - 1),
-            tile=data.shape[dim] if dim > 0 else batch_size,
-            dtype=np.int32,
-        )
-        for dim in range(data.ndim)
+@pytest.fixture
+def temp_uris():
+    """
+    Returns a temporary directory instance
+    """
+    return tempfile.TemporaryDirectory()
+
+
+@pytest.fixture(
+    params=[
+        {
+            "input_shape": (10,),
+            "hyperparams": ("categorical_crossentropy", "rmsprop", "accuracy"),
+        },
+        {
+            "input_shape": (10, 3),
+            "hyperparams": ("categorical_crossentropy", "rmsprop", "accuracy"),
+        },
+        {
+            "input_shape": (10, 10, 3),
+            "hyperparams": ("categorical_crossentropy", "rmsprop", "accuracy"),
```

Comment:
You are right. I basically left them cause I spend time hacking the parametrisation of `pytest.fixture` with two or more parameters and left it there in case of future use. No worries I will remove